server/job_service.go

```diff
@@ -314,14 +358,20 @@ func (p *Plugin) initJobService() error {
 	if !ok {
 		return fmt.Errorf("failed to get recorder version from manifest")
 	}
-	recordingJobRunner = "mattermost/calls-recorder:" + recorderVersion
+	recorderJobRunner = "mattermost/calls-recorder:" + recorderVersion
+
+	transcriberVersion, ok := manifest.Props["calls_transcriber_version"].(string)
+	if !ok {
+		return fmt.Errorf("failed to get transcriber version from manifest")
+	}
+	transcriberJobRunner = "mattermost/calls-transcriber:" + transcriberVersion
 
 	jobService, err := p.newJobService(p.getConfiguration().getJobServiceURL())
 	if err != nil {
 		return fmt.Errorf("failed to create job service: %w", err)
 	}
 
-	if err := jobService.Init(recordingJobRunner); err != nil {
+	if err := jobService.Init([]string{recorderJobRunner, transcriberJobRunner}); err != nil {
```

Comment:
Yeah but we are just passing the runner so that the docker image can be prefetched if needed. I think that's fine as transcriptions could be turned on at a later time without having to restart everything. But not super opinionated either.

Reply:
I just wonder if people will be confused by it prefetching the transcriber image (looking like it's getting ready for it) but they haven't enabled it? (just thinking, not necessary)

Reply:
Sure, I can get on board with the "don't do it if not needed" reasoning.